src/management/fs.c

```diff
@@ -32,6 +32,8 @@
 #include "internal/fs.h"
 #include "internal/error_handler.h"
 
+#define HASH_MAX_LEN	32
+#define FS_NAME_MAX_LEN	255
```

Comment:
Please delete the extra spaces.
Also, we can use the linux inbuild macro instead of defining a new one here.
`#define NAME_MAX         255    /* # chars in a file name */` defined in `/usr/include/linux/limits.h`

Reply:
Done